fvtest/sigtest/sigTest.cpp

```diff
@@ -69,7 +69,13 @@ enum TestAction {
 #endif /* defined(OMR_OS_WINDOWS) || defined(J9ZOS390) */
 #endif
 
+#if defined(FREEBSD)
+static sigjmp_buf env;
+static jmp_buf jenv;
+#else /* defined(FREEBSD) */
 static jmp_buf env;
+static jmp_buf jenv;
+#endif /* defined(FREEBSD) */
```

Comment:
Did you intend to push this? Looks like this isn't necessary and would have been fine the way it was before based on your usage below.

Reply:
Yes see https://www.freebsd.org/cgi/man.cgi?query=setjmp&sektion=3 "Pairs of calls may	be intermixed, i.e., both sigsetjmp() and siglongjmp() and setjmp() and longjmp() combinations may be used in the same program, however, individual calls may not,	e.g. the env argument to setjmp() may not be passed to siglongjmp()."

It appears this is not explicitly defined on Linux but I didn't check the implementation http://man7.org/linux/man-pages/man3/setjmp.3.html

Reply:
You're right, I didn't look at the rest of the file and didn't realize the existing Linux code was using both and passing a `jmp_buf` to `sigsetjmp`. The Linux version was and still is wrong isn't it? Shouldn't the BSD side of this `#ifdef` be used by both?

Reply:
@ymanton ok I can probably just remove the ifdef and have two trackers then